neqo-transport/src/qlog.rs

```diff
@@ -545,7 +545,7 @@ impl From<&Frame<'_>> for QuicFrame {
                 },
                 error_code: Some(error_code.code()),
                 error_code_value: Some(0),
-                reason: Some(String::from_utf8_lossy(reason_phrase).to_string()),
+                reason: Some(reason_phrase.to_string()),
```

Comment:
```suggestion
                reason: Some(reason_phrase),
```

Isn't `reason_phrase` a string now?

Reply:
Making this change required some surgery to `impl From<Frame<'_>> for QuicFrame`, please take a look.